src/Orleans/Serialization/SerializationManager.cs

```diff
@@ -2036,6 +2077,24 @@ public static void LogRegisteredTypes()
         }
         
         /// <summary>
+        /// Creates an instance of an external serializer
+        /// </summary>
+        /// <param name="type">The type that that implements <see cref="IExternalSerializer"/></param>
+        private static void RegisterExternalSerializer(Type type)
+        {
+            try
+            {
+                var serializer = Activator.CreateInstance(type) as IExternalSerializer;
```

Comment:
@jason-bragg I don't think we can do that since the serializer has to run on the GrainClient as well